pom.xml

```diff
@@ -168,6 +168,11 @@
             <artifactId>commons-csv</artifactId>
             <version>1.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-redis</artifactId>
+            <version>2.0.7.RELEASE</version>
+        </dependency>
```

Comment:
We already have spring-boot-starter-data-redis. why need this?

Reply:
Will remove and commit